meshtastic/stream_interface.py

```
@@ -32,6 +32,7 @@
     serial.SerialException,
     serial.SerialTimeoutException,
 )
+STREAM_CLOSE_EXCEPTIONS = STREAM_IO_EXCEPTIONS + (TypeError,)
```

Ruff (0.15.4)

[warning] Consider `(*STREAM_IO_EXCEPTIONS, TypeError)` instead of concatenation

Replace with `(*STREAM_IO_EXCEPTIONS, TypeError)`

(RUF005)